src/Components/Button.jsx

```diff
@@ -1,7 +1,32 @@
-const Calculator = () => (
-  <div>
-    calculator
+import './button.css';
+
+const Button = () => (
+  <div className="container">
+    <div className="grid-container">
+
+      <input className=" value-input" type="text" value="0" />
+      <button type="button">AC </button>
+      <button type="button">+/- </button>
+      <button type="button"> % </button>
+      <button type="button" className="grid-item-color">/ </button>
```

Comment:
- [ ] Kindly use the correct division sign on the design, &div;. We need to maintain the design as much as possible. If you do not have the sign on your keyboard, then kindly use HTML entity code for it.